### ISSUE TITLE
0.6.0 windows startup error

### ISSUE THREAD

**Issue by robert-phillips1** [huang-julien/nitro-applicationinsights#58]
Hello,

I'm getting an issue after following the migration steps for 0.6.0

windows 11
node 20.11.1
nuxt 3.11.1
nitro 2.9.4

![image](https://github.com/huang-julien/nitro-applicationinsights/assets/66948161/3fba0b2a-3488-4761-bf25-8c048a30cf0e)

**Comment by huang-julien** [comment]
Hi :wave: do you have a minimal reproduction please ? 
https://stackblitz.com/edit/nuxt-starter-qn4x7n

**Comment by robert-phillips1** [comment]
can confirm this workaround got me unblocked:

```
// package.json
  "overrides": {
    "mlly": "1.4.0"
  }
```

**Comment by huang-julien** [comment]
tracking in https://github.com/unjs/nitro/issues/2308

**Comment by huang-julien** [comment]
@robert-phillips1 can you also try with `experimental.legacyExternal` in your nitro config ? it would avoid you to force the resolution of mlly to 1.4.0

**Comment by huang-julien** [comment]
@robert-phillips1 with nitro-applicationinsights now being a nitro module (0.7.0) you shouldn't need to add any workaround anymore. The module will do it for you.

I'm closing this issue. If there's any issue again related to this, feel free to reopen it or to open another issue :)
